geo/src/test/java/org/odk/collect/geo/selection/SelectionMapFragmentTest.kt

```diff
@@ -165,11 +165,34 @@ class SelectionMapFragmentTest {
             .check(matches(isDisplayed()))
     }
 
+    @Test
+    fun `shows poly when item has multiple points`() {
+        val items: List<MappableSelectItem> = listOf(
+            Fixtures.actionMappableSelectItem().copy(
+                id = 0,
+                points = listOf(
+                    MapPoint(40.0, 0.0),
+                    MapPoint(41.0, 0.0)
+                )
+            )
+        )
+
+        val itemsLiveData = MutableLiveData(items)
+        whenever(data.getMappableItems()).thenReturn(itemsLiveData)
+
+        launcherRule.launchInContainer(SelectionMapFragment::class.java)
+        map.ready()
+
+        assertThat(map.getPolys(), equalTo(itemsLiveData.value?.map { it.points }))
+        onView(withText(application.getString(R.string.select_item_count, "Things", 0, 1)))
```

Comment:
Why it's 0 and 1? As I understand the first number represents the number of choices that exist and the second one is the number of displayed choices. Sometimes if there is something wrong with choices the displayed text could have 1 and 0 but not there another way around.

Reply:
The numbers load independently: the first from `SelectionMapData#getItemCount` and the second from the size of `SelectionMapData#getMappableItems`. In this test case, `SelectionMapData#getItemCount` is not being stubbed (or is just being stubbed with a default return value) as we're not concerned about it.